src/v2/view-builder/views/TerminalView.js

```diff
@@ -124,16 +135,18 @@ const Body = BaseForm.extend({
 });
 
 const Footer = BaseFooter.extend({
+  // All terminal views should have Back to sign in link either widget configured or server configured.
```

Comment:
e2e test for this

Reply:
yup following up on that.. can you review rest of the PR?

Reply:
+1 to ensuring each of these new errors have testcafe tests.

As a follow-up, when these terminal errors occur via embedded scenarios (enduser settings, step-up, etc), we'd like to honor the [`hideSignOutLinkInMFA`](https://github.com/okta/okta-signin-widget/blob/master/docs/interaction_code_flow.md#featureshidesignoutlinkinmfa) configuration option. Hopefully, we can look to combine all of our footer logic into one place.

Reply:
rest of the changes look good

Reply:
BaseFooter does that https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/internals/BaseFooter.js#L58

Reply:
Ah - you're right. Overlooked that we were extending on top of `BaseFooter` 😞 . Thanks for clarifying!